src/components/onboarding/OnBoardingModal/OnBoardingContentContainer.tsx

```diff
@@ -116,6 +136,46 @@ export default function OnBoardingContentContainer({
             />
           </MutedDiv>
         )}
+        {showSignerConfirmBtn && (
```

Comment:
if possible, could you move this logic to the Signer.tsx?
so what I'm thinking is, you create a new props for OnBoaridngContentContainer, named for example customContinueButton or customFooter maybe?